src/cipher/cipher.rs

```diff
@@ -147,11 +143,6 @@ impl PoseidonCipher {
         MESSAGE_CAPACITY
     }
 
-    /// Bytes consumed on serialization of the poseidon cipher
-    pub fn serialized_size() -> usize {
-        ENCRYPTED_DATA_SIZE
-    }
```

Comment:
This is also should be kept for the same reason of `ECRYPTED_DATA_SIZE`, sorry I missed the last time. I'm not sure if it's widely used, but I remember it is used as standard function for a lot of other types.

Also, I would suggest to convert to to a `const fn` (as we have for the other types).

Reply:
Done